brainglobe_registration/widgets/similarity_metrics_widget.py

```diff
@@ -0,0 +1,162 @@
+from qtpy.QtCore import (
+    Signal,  ## https://www.tutorialspoint.com/pyqt/pyqt_new_signals_with_pyqtsignal.htm
+)
+from qtpy.QtWidgets import (
+    QCheckBox,
+    QComboBox,
+    QGroupBox,
+    QHBoxLayout,
+    QLabel,
+    QPushButton,
+    QSizePolicy,
+    QSpinBox,
+    QVBoxLayout,
+)
+
+AVAILABLE_METRICS = ["ncc", "mi", "ssim"]  # from similarity metrics code
+
+
+class SimilarityWidget(QGroupBox):
+    calculate_metric_requested = Signal(object, object, str)
```

Comment:
I think we can make the types in the signal stronger, `object` is a little too open in my opinion. For signalling error cases, we can signal -1, or not pass a signal at all.

```suggestion
    calculate_metric_requested = Signal(int, int str)
```